templates/home/index.html.twig

```diff
@@ -1,7 +1,128 @@
 {% extends 'base.html.twig' %}
 
-{% block title %}{{ someVariable }}{% endblock %}
+{% block title %}{% endblock %}
 
 {% block body %}
-    <div class="nfq-academy">{{ someVariable }}</div>
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Studentai</li>
+        {% for key, project in projects %}
+            {% for students in project.students %}
+                <li class="list-group-item">
+                    <a href="/student?name={{ students }}&amp;project={{ key }}">{{ students }}</a>
+                    (
+                    <span class="badge">Mentorius</span> {{ project.mentors[0] }}
+                    )
+                </li>
+            {% endfor %}
+        {% endfor %}
+
+    </ul>
+
+    <div class="m-4">
+        <form action="/student" method="get">
+            <div class="input-group">
+                <input name="name" type="text" class="form-control" placeholder="Studentas">
+            </div>
+            <div class="input-group">
+                <input name="project" type="text" class="form-control" placeholder="Projektas">
+            </div>
+            <button type="submit" class="btn btn-success">Sužinoti vertinimą</button>
+        </form>
+    </div>
+
+    <ul class="list-group m-4">
+        <li class="list-group-item list-group-item-info">Projektai</li>
+        <li class="list-group-item">
+            <h3>Nedaryk pats</h3>
```

Comment:
Šituos reikėtų sugeneruoti pagal JSON failą

Plačiau:
* https://twig.symfony.com/doc/2.x/tags/for.html